bat/BLDJAR.BAT

```diff
@@ -19,24 +19,27 @@ pushd %~dps0..
 rem both the compiler javac.exe and jar builder jar.exe must be on sdk path
 rem note you can use current z390.jar with compatible java runtime with sdk
 
-rem unless suppressed, generate javadoc. Ignore any errors
-if /I %1. NEQ Nodocs. (call bat\BLDJVDOC.BAT
-               ) else (shift /1
-                       )
+rem process options
+set z_docs=Y
+set z_dbg=N
+:opt_loop
+if /I %1. EQU . goto options_ok
+if /I %1. EQU Nodocs.  set z_docs=N
+if /I %1. EQU docs.    set z_docs=Y
+if /I %1. EQU Debug .  set z_dbg=Y
+if /I %1. EQU Nodebug. set z_dbg=N
+shift /1
+goto opt_loop
+:options_ok
 
 rem compile java code and create jar file
 if not exist ".\jar\" mkdir jar
-
-erase jar\*.java
-erase jar\*.class
-erase jar\z390.jar
-copy  src\*.java   jar
-copy  src\z390.man jar
 cd jar
+
```

Comment:
Need to empty the entire jar directory. Either before the cd jar or after. Definitely before the javac. See bash/bldjar to see how it was done for linux.